autobahn/src/main/java/io/crossbar/autobahn/wamp/Session.java

```diff
@@ -445,8 +445,18 @@ private void onMessage(IMessage message) throws Exception {
                         msg.registration));
             }
 
+            long callerSessionID = msg.details != null && msg.details.get("caller") != null 
```

Comment:
The change looks good, thanks for doing it. Could you please change this code with a call to `getOrDefault()`, which is designed exactly for such cases.